SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  SKYChatConversationViewController.swift
+//  SKYKitChat
+//
+//  Copyright 2016 Oursky Ltd.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import JSQMessagesViewController
+
+class SKYChatConversationViewController: JSQMessagesViewController {
+    public var skygear: SKYContainer = SKYContainer.default()
+    public var conversation: SKYConversation?
+    public var participants: [String: SKYRecord] = [:]
+    public var messages: [SKYMessage] = []
+
+    static let errorDomain: String = "SKYChatConversationViewControllerErrorDomain"
+    var errorCreator: SKYErrorCreator {
+        return SKYErrorCreator(defaultErrorDomain: SKYChatConversationViewController.errorDomain)
+    }
+}
+
+// MARK: - Initializing
+
+extension SKYChatConversationViewController {
+
+    public class func create() -> SKYChatConversationViewController {
+        return SKYChatConversationViewController(nibName: "JSQMessagesViewController",
```

Comment:
Yes. I have tried that in the previous PRs. 

Overriding the designated initializer `init(nibName:, bundle:)` will cause the compiler complain that we also need to override another initializer `
init(coder:)` from `NSCoding` protocol. There will be much duplicated code for that. 

So, finally we chose to use to use a static method for specifying the nib.